### PR TITLE
Hooked up boxplot function to Shiny

### PR DESCRIPTION
**Pull request by rwdudley-usgs**
Seems to be working ok; though I noticed when turning on notches they don't render correctly. I didn't look into why yet.

### CODE REVIEW

**Comment by grrmartin-USGS** [comment]
It may be the small number of observations in some cases. DE and HI have
too few counties.  The plots are all  the state's area (county, huc,
aquifer, etc) values of a particular data element.

Gary Martin (grmartin@usgs.gov)

Hydrologist

U.S. Geological Survey

Kentucky Water Science Center

9818 Bluegrass Parkway

Louisville, Kentucky

40299-1906

(502) 609-1383 (cell)

(502) 493-1914 (phone)

(502) 493-1909 (fax)

File: inst/shiny/plotBoxplots.R [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  
+  data.elements <- c(df[["data.element"]],df[["data.element.y"]])
+  
+  areas.p2e <- df[["area"]]
+  
+  if(all(df[["areas"]] %in% areas.p2e)){
+    areas.p2e <- NA
+  }
+  notch <- input$notchOn
+  yrs <- NA # may want to code ability to select certain years
```
Comment (ldecicco-USGS) [inline inst/shiny/plotBoxplots.R:13]: I think this is fine. I've made a few decisions like this where the options on the app are a little more simplified than the package function. If you offer too many choices, the UI starts to get confusing.

**Comment by ldecicco-USGS** [comment]
I fixed a little conflict and will push it up now.

File: inst/shiny/plotBoxplots.R (hunk 1 above)
Comment (rwdudley-usgs) [inline inst/shiny/plotBoxplots.R:13]: Good. I initially tried including it, but quickly opted to go with NA to just get the new tab to work first! It was an interesting reverse-engineering process mimicking the stuff you'd already put together. Much of it I can follow, some of it I have to admit is still a mystery.

**Pull request closed without merging**